custom/opm/reports.py

```diff
@@ -864,7 +864,6 @@ class MetReport(CaseReportMixin, BaseReport):
     model = ConditionsMet
     exportable = False
     default_rows = 5
-    ajax_pagination = True
```

Comment:
We were already calculating all the rows anyways, then throwing away most of them.  This just sends them all down to the client so at least paginating is fast.